### ISSUE TITLE
Searching for username AND string on Search API not working for me.

### ISSUE THREAD

**Issue by philliptmaddox** [ttezel/twit#215]
I am trying to figure out how to search for the operator '@username' AND 'cheese pizza' (not literally). But I want to search a user's tweets by string. Submitting q: '@username' works and submitting q: 'cheese pizza' works but I can't seem to figure out how to combine this into one query.

**Comment by dominicrico** [comment]
You need to add the `from:` in front of the username and then just add the rest like: `from:@username cheese pizza`.

**Comment by philliptmaddox** [comment]
I'm still having issues. It seems like this query string 'works' but I'm only able to pull a single result from my last tweet only. It won't pull any more tweets than that regardless of keyword. I've tried sending q:'from:phillipmaddox' only and it doesn't work either. Just one tweet.

'''
// ACCESS INDIVIDUAL TIMELINE
var params = {q: 'from:phillipmaddox appointment until:2015-10-27', count: '5'}
twitter.get('search/tweets', params, function(error, tweets, response){
	if (!error) {
		console.log(tweets);	
	}
});
'''

**Comment by dominicrico** [comment]
Maybe this helps you. 

From the Twitter API docs:

#### until
##### optional
> Returns tweets created before the given date. Date should be formatted as YYYY-MM-DD. Keep in mind that the search index has a 7-day limit. In other words, no tweets will be found for a date older than one week'.
`Example Values: 2015-07-19`


So you see, if your tweets are older than one week they will not be displayed via the API!

**Comment by philliptmaddox** [comment]
I did not see that line (not surprisingly) before. However, changing 'until' to 'since' and backing 2015 to 2013 still returns nothing for the given keyword: appointment. I have used the word appointment in a tweet to Comcast recently so I know there should be a tweet with that. Still, even when I try other keywords it's only returning if I hit on one that existed in my very last tweet. 

Sorry to drag this out but I've been tinkering with it for over a while now!
